clippy.toml

```diff
@@ -0,0 +1,2 @@
+type-complexity-threshold = 710
```

Comment:
if we would like to use `clippy` config file, we should consider making it compatible with `makefile` because makefile should have already disabled some lints

Reply:
we shouldn't put configs in two places

Reply:
Ok, I will continue with the `make lint` configuration.